macros/public/bytea2numeric_v3.sql

```diff
@@ -0,0 +1,66 @@
+{% macro bytea2numeric_v3() %}
+    create or replace function bytea2numeric_v3(column_name STRING)
+    returns STRING
+    return
+    with base_factors as (
+        select
+         conv(substring(lpad(column_name::string, 64, '0'),1,8),16,10)::decimal(38) as hex_5        -- 16^56 -> 16^63
+        ,conv(substring(lpad(column_name::string, 64, '0'),9,14),16,10)::decimal(38) as hex_4      -- 16^42 -> 16^55
+        ,conv(substring(lpad(column_name::string, 64, '0'),23,14),16,10)::decimal(38) as hex_3     -- 16^28 -> 16^41
+        ,conv(substring(lpad(column_name::string, 64, '0'),37,14),16,10)::decimal(38) as hex_2     -- 16^14 -> 16^27
+        ,conv(substring(lpad(column_name::string, 64, '0'),51,14),16,10)::decimal(38) as hex_1     -- 16^0 -> 16^13
+        --10^17
+        ,pow(10,17)::decimal(38) as pow_10_17
+        --16^14
+        ,'72057594037927936'::decimal(38) as base_14_1
+        --16^28
+        ,substring('5192296858534827628530496329220096',1,17)::decimal(38) as base_28_2
+        ,substring('5192296858534827628530496329220096',18,17)::decimal(38) as base_28_1
+        --16^42
+        ,substring('374144419156711147060143317175368453031918731001856',1,17)::decimal(38) as base_42_3
+        ,substring('374144419156711147060143317175368453031918731001856',18,17)::decimal(38) as base_42_2
+        ,substring('374144419156711147060143317175368453031918731001856',35,17)::decimal(38) as base_42_1
+        --16^56
+        ,substring('26959946667150639794667015087019630673637144422540572481103610249216',1,17)::decimal(38) as base_56_4
+        ,substring('26959946667150639794667015087019630673637144422540572481103610249216',18,17)::decimal(38) as base_56_3
+        ,substring('26959946667150639794667015087019630673637144422540572481103610249216',35,17)::decimal(38) as base_56_2
+        ,substring('26959946667150639794667015087019630673637144422540572481103610249216',52,17)::decimal(38) as base_56_1
+    )
+
+    --compose all factors together in the correct decimal(38)s
+    , decimals as (
+    select
+         hex_input_64
```

Comment:
where should this column come from? it's not defined in `base_factors`

Reply:
Thanks, I removed this column, it's not needed and was a relic from exploration.